### PR TITLE
Add Swift Package Manager support

### LINKED ISSUE #2: Migrate to Swift 5.1 and add SwiftPM support
- Migrate framework to Swift 5.1
- Add SwiftPM support

### PR DESCRIPTION
**Pull request by cipi1965**
This PR fixes #2

### CODE REVIEW

**Comment by pedrommcarrasco** [comment]
Thank you for contributing @cipi1965! 👍 
Could you also update the `README.md` with SwiftPM?

**Comment by pedrommcarrasco** [comment]
Also, please bump the Swift Version to `5.1` in the project and fix any warning that shows up,

File: Fluky/Fluky/Views/FlukyImageView.swift [hunk 1]
```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2018 Pedro Carrasco. All rights reserved.
 //
 
-import Foundation
```
Comment (pedrommcarrasco) [inline Fluky/Fluky/Views/FlukyImageView.swift:9]: Nice job!

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// swift-tools-version:5.0
```
Comment (pedrommcarrasco) [inline Package.swift:1]: This should be 
`// swift-tools-version:5.1`

File: Package.swift [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "Fluky",
+            targets: ["Fluky"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
```
Comment (pedrommcarrasco) [inline Package.swift]: Please delete all these comments

File: Package.swift [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(
+            name: "Fluky",
+            dependencies: [],
+            path: "Fluky/Fluky"),
+    ],
+    swiftLanguageVersions: [
```
Comment (pedrommcarrasco) [inline Package.swift:21]: Do we need this?

**Comment by cipi1965** [comment]
@pedrommcarrasco is there any reason to bump version to Swift 5.1 and not support 5.0?

**Comment by pedrommcarrasco** [comment]
It's the latest version of Swift with the latest features. We're not an huge framework so there's not much reasons to avoid keeping it updated @cipi1965.

If you don't agree, feel free to keep it at Swift 5.0. Just update the project Swift Version and validate that there are no warnings

**Comment by cipi1965** [comment]
@pedrommcarrasco Swift 5.1 is a minor update, i think it's not needed to require that version instead of 5.0

File: Package.swift (hunk 4 above)
Comment (cipi1965) [reply Package.swift:21]: What? Do you mean empty dependencies array?

**Comment by pedrommcarrasco** [comment]
Looks good to me @cipi1965! Once again, thanks for taking the initiative! Enjoy the rest of the Hacktoberfest! 🙌

**Comment by cipi1965** [comment]
@pedrommcarrasco Thank you!

**Pull request merged**